src/bootstrap/test.rs

```diff
@@ -2217,6 +2217,11 @@ fn prepare_cargo_test(
 ) -> Command {
     let mut cargo = cargo.into();
 
+    // If bless is passed, give downstream crates a way to use it
+    if std::env::args().any(|arg| arg == "--bless") {
```

Comment:
There is a `bless` flag you can check: `builder.config.cmd.bless()`

Reply:
This actually came up on Zulip the other day, Clippy and Miri kind of wanted to unite their variables that do the same thing. I did that change in #113298 and requested you for review.

I'll just checkout the relevant parts of that to this one so there's no conflict.

Reply:
I take this back, there wound up being some more refactoring in that that I don't want to pull into here. So I just made the change you suggested, I can adjust the other PR if this lands first. (should be good to go here agaiin)